src/platform/riot/uart_channel.c

```diff
@@ -0,0 +1,161 @@
+#include "reactor-uc/platform/riot/uart_channel.h"
+#include "reactor-uc/logging.h"
+#include "reactor-uc/environment.h"
+#include "reactor-uc/serialization.h"
+#include "led.h"
```

Comment:
Is this needed?

Reply:
I think yes

Reply:
What is the LED used for, I cant see any code that uses something LED related?